### ISSUE TITLE
Theme: Cloned modules

### ISSUE THREAD

**Issue by koumdros** [qooob/authentic-theme#500]
I've noticed that if I clone a module, the new cloned instance, never appears in authentic theme.

If I change to another theme( tried the original grey them + caldera) , it appears normally, And if I edit it and switch back to Authentic it picks it up.

to be specific, I added a clone of apache, to configure for an apache 2.4 version ( via software collections) , and saw this happening. I also saw it with 2 different custom versions of MySQL.

I dont think it should matter, just mentioning it for completeness.

**Comment by qooob** [comment]
Did you try reloading the theme (left menu) after module was cloned?

**Comment by koumdros** [comment]
I tried relaoding.
I tried stopping Webmin and starting again. 
I tried changing to another theme and back to authentic. 
none of the above worked, sorry I should have submitted this info in my original post. ( in my ovewrkrked mind it was "obvious" I did all that, which it wasnt of course...

**Comment by qooob** [comment]
> And if I edit it and switch back to Authentic

Could you specify, what exactly do you mean here?

**Comment by koumdros** [comment]
I mean if I clone the theme, then go into the theme while working under "grey" theme, and change some options, save them, and then switch to authentic-theme, THEN it picks it up.

**Comment by qooob** [comment]
>I mean if I clone the theme
Theme or module??

When you edit/save settings Webmin is reastarted and this is why it might work.

Try to restart Webmin after cloning module without changing theme, does it work then?

**Comment by koumdros** [comment]
Sorry for not being clear.
1.  I cloned a module , not a theme.
   * this heppened with : Apache, mySQL and PostgresQL modules
2. I did try restarting webmin after cloning
 * also tried stopping and starting ( tis houldnt make any difference.. but_)

nothing worked so far

**Comment by qooob** [comment]
Did you try going to ACL settings (clicking user icon on the navigation menu, near log-out button) and checking if newly added/cloned modules have enabled permissions?

How this issue related to the theme? :)

**Comment by koumdros** [comment]
No I didn't actually, but I guessed it must have been relevant cause in the
other themes , newly cloned themes did appear. I am currently on mobile
I'll check and get back to you.

N.

**Comment by koumdros** [comment]
ok, back. 
Since the user in question is actually root ( that;s the one I did the cloning with..), isnt he suposed to have acces to all modules?
Nevertheless, I went to the ACL settings, the cloned module does appear there but it doesnt appear in the modules list. 

I;m repeating all steps in 3 diferent servers, all almost identical in setup.: 
A. cloned module, picked "reset settings"
B. module doesnt appear in either "Servers" submenu, or in the Un-used modules ( where it usually  appears in other themes..)
C. Checked ACL lists, the cloned module IS present.
D. loged via ssh, checked /etc/webmin the folder for the new module is created  but settings are identical ( ie not reset)
D . checked /usr/webmin/libexec theer is now a symlink for the module pointing to the original module folder ( as it should)
e. tried to refresh module list , this is the way I know to force webmin to "realise" there is a new module. it doenst work
F. opened webmin log + error log, nothing appears there..
G. restarted webmin.
H. 1 server out fo 3 the module appears! / 2 out of 3 it doesnt.
I. in those two servers I switch to a different theme ( Gray farmed theme.(
immediately the cloned theme appears...
J. Switch back to authentic theme. the module appears normally in all 3 servers.

Before now I've tried in ~6-7 different servers, this above was the first time a cloned module appeared in authentic...
